modules/runtime/nuxeo-connect-standalone/src/main/java/org/nuxeo/connect/update/task/guards/PlatformHelper.java

```diff
@@ -81,7 +79,7 @@ public boolean isJetty() {
      */
     @Deprecated
     public boolean isWindows() {
-        return System.getProperty("os.name").toLowerCase().indexOf("win") >= 0;
+        return System.getProperty("os.name").toLowerCase().contains("win");
```

Comment:
Unrelated to your changes but `toLowerCase` without a locale is wrong. This will fail in a Turkish default locale. I'd just do `System.getProperty("os.name").contains("Windows")`. Or just use Apache Lang `SystemUtils.IS_OS_WINDOWS`